boards/samd10-xmini/Makefile.dep

```diff
@@ -0,0 +1,10 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+# Use Picolibc to reduce ROM usage
+FEATURES_REQUIRED += picolibc
+USEMODULE += picolibc
```

Comment:
Shouldn't a used feature of `picolib` automagically pull in the corresponding module? It works like this e.g. for all `periph_%` features.

Reply:
That's an ugly hack I had to copy from `nucleo-l011k4`:

 - with only `FEATURES_REQUIRED += picolibc` [this check](https://github.com/RIOT-OS/RIOT/blob/master/cpu/cortexm_common/Makefile.dep) will on first iteration add the `newlib` module
- with only `USEMODULE += picolibc` `FEATURES_BLACKLIST` does not work.